src/harness/testcases/registration_testcase.py

```diff
@@ -128,3 +128,61 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+@winnforum_testcase
+  def test_WINFF_FT_S_REG_23(self):
+    """CBSD Cat A attempts to register with HAAT >6m
+
+    The response should be FAILURE.
+    """
+
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    device_a['installationParam']['latitude'] = 38.882162
+    device_a['installationParam']['longitude'] = 77.113755
+    device_a['installationParam']['height'] = 8
+    device_a['installationParam']['heightType'] = 'AGL'
+    device_a['installationParam']['indoorDeployment'] = False
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 202)
+    self.assertTrue('cbsdId' in response)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_24(self):
```

Comment:
The WINNF-15-s-0061 v0.6.0 no longer has INVALID_VALUE (103) in the test case description, although it was in v0.5.x. v0.6.0: 

SAS rejects the request by sending a CBSD Registration Response as follows:
* SAS response does not include cbsdId
* The responseCode in Response Object should be 202, concluding a failed registration.

Have sent an email to the spec author to clarify or update spec.